docs/evaluation/examples.rst

```diff
@@ -149,15 +149,18 @@ is used to show the grid used by
 
 .. _model_evaluate_example_pha:
 
-Handling PHA datasets
-=====================
-
-:term:`PHA` data is more complicated than other data types in Sherpa because
-of the need to convert between the units used by the model (energy
-or wavelength) and the units of the data (channel). As a user you
-will generally be thinking in keV or Angstroms, but the
-:py:class:`~sherpa.astro.data.DataPHA` class has to convert to
-channel units internally.
+X-ray data (`~sherpa.astro.data.DataPHA`)
```

Comment:
This doesn't create a link and so should just probably be `(sherpa.astro.data.DataPHA)` or removed completely - see https://sherpa--1182.org.readthedocs.build/en/1182/evaluation/examples.html#x-ray-data-sherpa-astro-data-datapha